### DISCUSSION TITLE
采用Railway平台部署，显示的时间与实际时间不一致

### DISCUSSION THREAD

**Discussion by junhuame** [walinejs/waline#1177] [Q&A] [answered]
请教一下，我部署完Waline后，评论显示的时间与实际时间不一致，显示的时间比北京时间快8小时，但是Railway平台的数据库查询到的评论时间是正确的北京时间，我也删除重新安装过还是不行。请问有没有遇到这样的问题，有没有解决方法。，谢谢~

![微信图片编辑_20220707232956](https://user-images.githubusercontent.com/22878607/177820366-2803ae9a-48ce-4033-a19a-c32c6f6ece00.jpg)
![微信图片编辑_20220707235937](https://user-images.githubusercontent.com/22878607/177820365-fbd07b44-d4dd-4d7e-a09f-fd4c6c5a6fac.jpg)

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/commit/a7f357748345f846160475f6e820f222d2f26837 按照这个 commit 修改下 pg 表的结构吧，把 `with time zone` 修改成 `without time zone`

**Reply by junhuame** [reply]
您好，我使用的是railway-starter，在这个库里没有看到表结构的配置文件；在https://railway.app/中也没找到配置文件，请问可以详细说一下吗，多谢了~

**Reply by lizheming** [reply]
直接修改 postgresql 的数据库字段，如果不会的话可以考虑直接重装~

**Reply by junhuame** [reply]
之前重装过不行，我研究一下修改字段，多谢
